analysis/deep2_r23_o32/archives/binning/binning_and_graphing_mastergrid.py

```diff
@@ -9,22 +9,24 @@
 def n_times_binned(fitspath, pdf_pages, outfile, n_split, individual_ID, R23, O32,
                    SNR3, data3, galinbin):
     """
-    R23 and O32 are going to be log values
-    One_dimensional binning for R23 followed by each bin being split in O32 in n_split bins
-    increase the number of galaxies as R23 increases
+    Purpose:
+      R23 and O32 are going to be log values
+      One_dimensional binning for R23 followed by each bin being split in O32 in n_split bins
+      increase the number of galaxies as R23 increases
 
-    This file holds the function to bin data adaptively based on the entered number of galaxies for each bin.
-    First, it bins in the R23 direction and then in the O32 direction
-    Used in current analysis
+      This file holds the function to bin data adaptively based on the entered number of galaxies for each bin.
+      First, it bins in the R23 direction and then in the O32 direction
+      Used in current analysis
 
     Inputs:
-    fitspath  -> path where files are called from and saved to
-    dataset   -> gives the name of the analysis being run
-    pdf_pages -> name of outputted pdf file
-    outfile   -> name of the npz file produced by the function
-    galinbin  -> array of numbers that specifies how many spectra go in each bin
-    Other variables -> emission file values of spectra that come from the get_det3 function
+      fitspath  -> path where files are called from and saved to
+      dataset   -> gives the name of the analysis being run
+      pdf_pages -> name of outputted pdf file
+      outfile   -> name of the npz file produced by the function
+      galinbin  -> array of numbers that specifies how many spectra go in each bin
+      Other variables -> emission file values of spectra that come from the get_det3 function
```

Comment:
:param fitspath: str. path where files are called from and saved to
:param dataset: str. gives the name of the analysis being run
:param pdf_pages: str. name of outputted pdf file
:param outfile: str. name of the npz file produced by the function
:param galinbin: array of numbers that specifies how many spectra go in each bin
Other variables: emission file values of spectra that come from the get_det3 function

Reply:
I haven't been consistent in how I write the parameters so as I am editing, I will try to update to fit this format.

Reply:
I rather have us do that later. This is focused on a merge conflict between your branch and this branch for logging. Let's get this in order otherwise it's going to be really confusing.

Reply:
I created #64 for this issue. Will address outside this PR.